books/RayTracingTheRestOfYourLife.html

```diff
@@ -650,12 +650,12 @@
 $I = 2.666… = \frac{8}{3}$. You could rightly point to this example and say that the integration is
 actually a lot less work than the Monte Carlo. That might be true in the case where the function is
 $f(x) = x^2$, but there exist many functions where it might be simpler to solve for the Monte Carlo
-than for the integration, like $f(x) = sin^5(x)$.
+than for the integration, like $f(x) = \sin^5(x)$.
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
         for (int i = 0; i < N; i++) {
             auto x = random_double(a, b);
-            sum += pow(sin(x), 5.0);
+            sum += pow(std::sin(x), 5.0);
```

Comment:
Should `pow` from <cmath> also use explicit std::?

Reply:
Good spot; I'll add it.